config.yaml

```diff
@@ -13,8 +13,174 @@ environment:
     auto_detect: True
 weather:
     station: mongo
+    aat_metdata:
+      name: 'AAT metdata'
```

Comment:
FYI, you don't actually need the quotes in YAML and then it looks a little cleaner.

Reply:
Okay, is that the same for the quotes that are in the array?